.shellcheckrc

```diff
@@ -0,0 +1,3 @@
+external-sources=true
+disable=SC2250
+source-path=dev-scripts/doc-fetches
```

Comment:
What is this file?

Reply:
It configured the shellcheck linter, which I'd definitely recommend installing into your VSCode.